### PR TITLE
satsuma new recipe

### PR DESCRIPTION
**Pull request by mictadlo**
* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/satsuma/build.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+cmake -D CMAKE_INSTALL_PREFIX:PATH=$PREFIX .
+make
+mkdir -p $PREFIX/bin
```
Comment (bgruening) [inline recipes/satsuma/build.sh]: is `make install` not available?

File: recipes/satsuma/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  url: https://github.com/bioinfologics/satsuma2/archive/da694aeecf352e344b790bea4a7aaa529f5b69e6.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [osx]
+
+requirements:
+  build:
+    - gcc_linux-64  # [linux64]
```
Comment (bgruening) [inline recipes/satsuma/meta.yaml]: The normal `gcc` package is not working?

Comment (mictadlo) [reply recipes/satsuma/meta.yaml]: gcc is too old.

File: recipes/satsuma/build.sh (hunk 1 above)
Comment (mictadlo) [reply recipes/satsuma/build.sh]: I tried `make install` but it failed.

**Pull request merged**